IfWrapper/Common/i2c_common.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file
+ * @brief I2C 通信関連の一般 IO 以外の公開関数
+ * @note レジスタの読み書き等，ハードウェアに依存しない関数群を定義する
+ */
+
+#ifndef I2C_COMMON_H_
+#define I2C_COMMON_H_
+
+#include "../../Drivers/Super/driver_super.h"
+#include "../i2c.h"
+
+/**
+ * @brief I2C_Config 構造体にて指定されたデバイスのレジスタへ 1 バイトデータを書き込みます
```

Comment:
```suggestion
 * @brief I2C_Config 構造体にて指定されたデバイスのレジスタへ 1 バイトデータを書き込む
```

しょうもないんですが，他でデスマス調つかってないので，ここの４箇所，直してもらえると助かります